### ISSUE TITLE
screen reader accessibility

### ISSUE THREAD

**Issue by RichCaloggero** [surge-synthesizer/releases-xt#2]
After installing surge-xt 64 bit on win7, using NVDA:

- when window opened it did not gain focus
   + set focus to first object in window when it initially opens
   + similarly for any child windows like dialogs, etc
- each control must be programmatically labeled
- tab should move among objects and cause each to gain focus

Latest Juce should have some of these accessibility features built in, but if creating your own controls then might need to implement yourself. 
 
Hopefully this is enough to get started on making this a synth that all users, regardless of disability, can use!

**Comment by baconpaul** [comment]
Hi @RichCaloggero and thanks for your comment. We've been tracking our accessibility work https://github.com/surge-synthesizer/surge/issues/4616 and so let me close this issue in favor of that one

But I'm surprised to hear your report. About 90% of the controls in surge are properly accessible, support a custom tab order, and work under automation on my mac, and our testers have reported them working on win10 NVDA. Except for modulation and a couple of our dialogs, everything has a correct programatic label and supports events, and our menus are all properly labeled too. So I'm surprised you had the experience you did (but the focus-on-open point is a great one and I added that to our list).  Could it be that Win7 support is different than Win10 perhaps?

But that is probably better discussed over on the larger issue.

Thanks so much!
